app/Http/Controllers/ResultsController.php

```diff
@@ -94,10 +94,12 @@ private function extractItemIds(LazyCollection $mismatches, array $datatypes): a
         $idsAsKeys = [];
 
         foreach ($mismatches as $mismatch) {
-            $idsAsKeys[$mismatch->item_id] = null;
+            if ($mismatch->wikidata_value !== '' && $mismatch->statement_guid !== '') {
```

Comment:
I think we still need the item label even if the value is missing, right? The condition should probably be only on the inner if. (Also, I find it confusing to test the `statement_guid` here – `wikidata_value` should be the only thing we care about IMHO.)